extensions/functions_comparison.yaml

```diff
@@ -70,6 +73,9 @@ scalar_functions:
     description: Whether a value is not null.
     impls:
       - args:
+          - name: nan_is_null
+            options: [ TRUE, FALSE ]
+            required: false
           - value: any1
         return: BOOLEAN
         nullability: DECLARED_OUTPUT
```

Comment:
This function is rather redundant. It's just `not(is_null(x))`. What's the reason for having both?